### PR TITLE
Migrating from Nexmo to Vonage lib

### LINKED ISSUE #9: Update project to use the `vonage` gem
This project uses the `nexmo` gem, but it should be updated to use the new `vonage` Ruby SDK gem: https://github.com/vonage/vonage-ruby-sdk. 

The `Gemfile` and code will need to be updated accordingly.

Bonus: Add tests to validate the success of the refactor! 🌟

### PR DESCRIPTION
**Pull request by rafaeelaudibert**
Closes #9

### CODE REVIEW

**Review by martyndavies — CHANGES_REQUESTED** [review]
This is a great PR. Please can you take a look at this change.

**Comment by laurenelee** [comment]
Lorna and I discussed this issue and highlighted what a great PR it was on the stream yesterday!

File: app/controllers/telephone_controller.rb [hunk 1]
```diff
@@ -1,77 +1,80 @@
...
         ].to_json
     end
 
     def event
         if params['recording_url']
             # Save Recording
-            puts "Saving Audio File"
-            NexmoClient.files.save(params['recording_url'], 'recording.wav')
+            puts 'Saving Audio File'
+            VonageClient.save_file(params['recording_url'], 'recording.wav')
```
[the remark this answers is not in the document]
Comment (rafaeelaudibert) [reply app/controllers/telephone_controller.rb:59]: Sure! Thanks for noticing it

**Comment by rafaeelaudibert** [comment]
Just applied the change required, @martyndavies! Thanks for noticing it.

Oh, and thanks for the citing on the livestream @laurenelee!

Going forward after this PR, I think it would make sense to change the name of the repository, dont u folks think?

**Review by martyndavies — APPROVED** [review]

**Pull request merged**